scripts/ec2cleanup.py

```diff
@@ -46,10 +46,12 @@ def delete_old_instances(texts, ec2provider, provider_key, date,
             report.write("\n{}:\n-----------------------\n".format(provider_key))
             for instance in ec2provider.list_vm(include_terminated=True):
                 creation = ec2provider.vm_creation_time(instance)
-                print("EC2:{}  {}  \t {}".format(
-                    provider_key, instance, (date - creation)))
-                report.write("EC2:{}  {}  \t {}\n".format(
-                    provider_key, instance, (date - creation)))
+                print("EC2:{}  {}  \t {} \t {} \t {}".format(
+                    provider_key, instance, (date - creation),
+                    ec2provider.vm_type(instance), ec2provider.vm_status(instance)))
+                report.write("EC2:{}  {}  \t {} \t {} \t {}\n".format(
+                    provider_key, instance, (date - creation),
+                    ec2provider.vm_type(instance), ec2provider.vm_status(instance)))
```

Comment:
a) I'm not sure but ```ec2provider.vm_type``` and ```ec2provider.vm_status``` could be expensive
b) the message is the same in both cases, so why not.....

```python
                message = "EC2:{}  {}  \t {} \t {} \t {}".format(
                    provider_key, instance, (date - creation),
                    ec2provider.vm_type(instance), ec2provider.vm_status(instance))
                print(message)
                report.write(message)
```